Added eye button for password view - Both login and signup

pranavsuriya-sr opened the pull request:
Added eye button for password view - Both login and signup.
<img width="500" height="700" alt="image" src="https://github.com/user-attachments/assets/1c709b68-c4ad-4ae9-97d0-e6d0f5a5e31b" />
<img width="500" height="700" alt="image" src="https://github.com/user-attachments/assets/8335c64f-6fd5-4893-87d3-7de5495d8da5" />
<img width="500" height="700" alt="image" src="https://github.com/user-attachments/assets/799d1196-1334-4952-9459-dc7a23f5d5c5" />

NotShrirang commented on package-lock.json:
```diff
@@ -10,6 +10,7 @@
       "dependencies": {
         "@tailwindcss/vite": "^4.1.13",
         "axios": "^1.12.2",
+        "lucide-react": "^0.544.0",
```
Instead of using Lucid React, can you use React Icons? It is already installed dependency.

NotShrirang commented on src/pages/RegisterPage.jsx:
```diff
@@ -1,6 +1,7 @@
 import { useState } from "react";
 import { Link, useNavigate } from "react-router-dom";
 import { useMyContext } from "../context/MyContext";
+import { Eye, EyeOff } from "lucide-react";
```
You can use

```js
import { FiEye, FiEyeOff } from "react-icons/fi";
```

Docs: https://react-icons.github.io/react-icons/search/#q=eye

pranavsuriya-sr commented:
Hi @NotShrirang. Yes i did it....Removed lucide react icons and added react-icons/fi....removed lucide react from package.json and package-lock.json as well. Kindly review the changes and merge it.

NotShrirang reviewed (approved):
Looks perfect
